apps/web-giddh/src/app/shared/advance-receipt-adjustment/advance-receipt-adjustment.component.html

```diff
@@ -131,19 +131,19 @@ <h4 class="modal-title bg pull-left">Adjust Payment</h4>
                             </div>
                         </div>
                     </ng-container>
-                    <span class="text-light" *ngIf="isUpdateMode">
+                    <span class="text-light" *ngIf="exceedDueAmount<0">
                         Excess amount:
-                        {{invoiceFormDetails.accountDetails.currencySymbol}}{{adjustPayment.totalAdjustedAmount- adjustPayment.grandTotal - depositAmount + adjustPayment.tcsTotal + adjustPayment.tdsTotal  | giddhCurrency}}
+                        {{invoiceFormDetails.accountDetails.currencySymbol}}{{-exceedDueAmount  | giddhCurrency}}
```

Comment:
Why additional '-' is used to display if the block will only be visible when `exceedDueAmount ` is negative

Reply:
Actually exceedDueAmount get -ve when adjust amount is greater than due amount so here we are showing -ve amount as positive